package.json

```diff
@@ -61,13 +61,13 @@
 		"shorten-repo-url": "^2.0.0",
```

Comment:
How home mem wasn't updated? There's 6.1.0

I generally use to check what's available and update all sub-dependencies correctly. npm still can't be trusted with the lock

```sh
npx npm-check -u && rm -rf node_modules package-lock.json && npm install && npm run test
```

Reply:
You said you did not want to update it

>I actually had left this on 6.0.1 because it's technically a better version since we don't use `mem.clear`
>
>_Originally posted by @fregante in https://github.com/sindresorhus/refined-github/pull/3168#discussion_r434486301_

Reply:
Yeah, because I thought that 6.1.0 was the most recent feature, but the "worse version" still hasn't been published yet https://github.com/sindresorhus/mem/pull/58#issuecomment-679381086

Reply:
Lock it @ 6.1.0?